appliances/Trivago/ebb81aaf055247c19579723cc84b8e74.yaml

```diff
@@ -0,0 +1,51 @@
+id: ebb81aaf055247c19579723cc84b8e74
```

Comment:
Can you please remove the **id** from YAML, it's taken from the filename...
I have updated documentation not to mention **id** already.